<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>program</string>
  </property>
  <property name="styleSheet">
   <string notr="true">ScrollProgressBar #backgroundFrame {
	background-color: gray;
}

ScrollProgressBar #progressFrame {
	background-color: red;
}

ProgressCircleBar &gt; #knobFrame {
	background-color: red;
}

ScrollProgressBar #progressFrame {
	background-color: red;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="action_play">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="action_skip">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>skip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="action_shuffle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>shuffle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="action_loop">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>loop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="action_loadFromURL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>load from url:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_playlistURL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="info_nowPlaying">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>now playing:</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_loadedPlaylist">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>loaded playlist:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="action_previous">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>previous</string>
    </property>
   </widget>
   <widget class="ScrollProgressBar" name="info_progressBar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="ProgressCircleBar" name="info_progressBar1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressCircleBar</class>
   <extends>ScrollProgressBar</extends>
   <header>customwidgets.scrollprogressbar.progresscirclebar</header>
  </customwidget>
  <customwidget>
   <class>ScrollProgressBar</class>
   <extends>QFrame</extends>
   <header>customwidgets.scrollprogressbar.scrollprogressbar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
